src/main/java/io/jenkins/plugins/designlibrary/Home.java

```diff
@@ -45,4 +47,13 @@ public UISample getDynamic(String name) {
         }
         return null;
     }
+
+    public String getPluginVersion() {
+        Jenkins jenkins = Jenkins.get();
+        if (jenkins != null) {
+            PluginWrapper plugin = jenkins.getPluginManager().getPlugin("design-library");
+            return plugin.getVersion();
```

Comment:
`getPlugin("design-library");` might return `null` (at least in theory), so you need to to add an `if (plugin != null)` guard here.